nui/assets/script.js

```diff
@@ -8,6 +11,7 @@ let maxNotifications;
 
 // This is where we store persistent noti's
 const persistentNotis = new Map();
+const RESOURCE_NAME = window.invokeNative ? window.GetParentResourceName() : 't-notify'
```

Comment:
```suggestion
const RESOURCE_NAME = !isBrowserEnv ? window.GetParentResourceName() : 't-notify'
```

Reply:
Pretty sure you meant to call the function.